src/SiteMapperService.ts

```diff
@@ -0,0 +1,36 @@
+import Sitemapper from 'sitemapper'
+import {getLogger} from "./logger";
+import {join} from "path";
+
+const outDir = join(process.cwd(), "bl-tmp")
+const quiet = true
+const logger = getLogger({ outDir, quiet });
+
+
+
+export const retrieveSites = async (url: string): Promise<any[]> => {
```

Comment:
Isn't this type more accurate?

```suggestion
export const retrieveSites = async (url: string): Promise<{href: string}[]> => {
```